src/_nebari/stages/infrastructure/__init__.py

```diff
@@ -520,6 +522,18 @@ def _check_input(cls, data: Any) -> Any:
                     raise ValueError(
                         f"Amazon Web Services instance {node_group.instance} not one of available instance types={available_instances}"
                     )
+
+        # check if eks cluster endpoint access config is valid
```

Comment:
This validator could be removed if you switch to an enum with a default value

Reply:
Removed validator per your suggestion; using `typing.Literal`, in lieu of pydantic enums, for congruency with pending [PR#2668](https://github.com/nebari-dev/nebari/pull/2668)'s invocation of [Literal](https://github.com/nebari-dev/nebari/blob/2603-aws-node-launch-template/src/_nebari/stages/infrastructure/__init__.py#L9)